lib/storages/sequelize.js

```diff
@@ -56,6 +53,7 @@ module.exports = redefine.Class({
     if (!this.options.storageOptions.model) {
       var sequelize = this.options.storageOptions.sequelize;
       var modelName = this.options.storageOptions.modelName;
+      var columnType = this.options.storageOptions.columnType || sequelize.constructor.STRING;
```

Comment:
Could you do me a favor and add a line `var Sequelize = sequelize.constructor` ?